core/src/main/java/com/vml/es/aem/acm/core/script/ScriptScheduler.java

```diff
@@ -61,10 +65,36 @@ public class ScriptScheduler implements Runnable {
 
     private Config config;
 
+    private long intervalBetweenRuns = -1;
+
     @Activate
     @Modified
     protected void activate(Config config) {
         this.config = config;
+        intervalBetweenRuns = calculateInterval();
+    }
+
+    private long calculateInterval() {
+        try {
+            CronExpression expression = new CronExpression(this.config.scheduler_expression());
+            Date now = new Date();
+            Date nextRun = expression.getNextValidTimeAfter(now);
+            if (nextRun == null) {
+                return -1;
```

Comment:
nextRun can be even null? if so cron will be not cron :) please do not check things that will never happen